modules/cms-kit/src/Volo.CmsKit.Domain/Volo/CmsKit/Blogs/CmsKitBlogOptions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace Volo.CmsKit.Blogs
+{
+    public class CmsKitBlogOptions
+    {
```

Comment:
Why do we need this type?

Reply:
It's not necessary anymore, I've just removed it 👍